### PR TITLE
Add endpoints to administer navaids

### PR DESCRIPTION
**Pull request by AxonC**
As a predecessor to administer holds, this PR adds endpoints to adminster Navaid models, which are later related to a hold.

### CODE REVIEW

File: app/Http/Requests/NavaidRequest.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+class NavaidRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
```
Comment (AndyTWF) [inline app/Http/Requests/NavaidRequest.php:17]: Do we always want this to be true?

File: tests/app/Models/Hold/HoldModelTest.php [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+use App\Models\Navigation\Navaid;
+
+class HoldModelTest extends BaseFunctionalTestCase
+{
+    public function testRouteKeyIsIdentifierColumn()
+    {
+        $model = new Navaid();
+        $this->assertEquals($model->getRouteKeyName(), 'identifier');
+    }
+}
```
Comment (AndyTWF) [inline tests/app/Models/Hold/HoldModelTest.php:15]: Newline

File: tests/app/Http/Controllers/Admin/NavaidAdminControllerTest.php [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    public function testNavaidCanBeDeleted()
+    {
+        Navaid::factory()->create(['identifier' => $this->identifier]);
+
+        $response = $this->makeAuthenticatedApiRequest("DELETE", "{$this->baseEndpoint}/{$this->identifier}");
+
+        $response->assertStatus(204);
+        $this->assertDatabaseMissing('navaids', ['identifier' => $this->identifier]);
+    }
+}
```
Comment (AndyTWF) [inline tests/app/Http/Controllers/Admin/NavaidAdminControllerTest.php:172]: Newline

File: app/Http/Requests/NavaidRequest.php (hunk 1 above)
Comment (AxonC) [reply app/Http/Requests/NavaidRequest.php:17]: The auth should be handled by the middleware and OAuth scopes so I'd say so

File: tests/app/Http/Controllers/Admin/NavaidAdminControllerTest.php (hunk 3 above)
Comment (AxonC) [reply tests/app/Http/Controllers/Admin/NavaidAdminControllerTest.php]: sorted

File: tests/app/Models/Hold/HoldModelTest.php (hunk 2 above)
Comment (AxonC) [reply tests/app/Models/Hold/HoldModelTest.php]: sorted

File: tests/app/Http/Controllers/Admin/NavaidAdminControllerTest.php (hunk 3 above)
Comment (AxonC) [reply tests/app/Http/Controllers/Admin/NavaidAdminControllerTest.php]: sorted

File: app/Http/Requests/NavaidRequest.php (hunk 1 above)
Comment (AndyTWF) [reply app/Http/Requests/NavaidRequest.php:17]: Sweet

**Review by AndyTWF — APPROVED** [review]

**Pull request merged**